lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -43,9 +43,10 @@
                 required: true
             host_group:
                 description:
-                    - Host group will be used for searching hosts.
+                    - Host group(s) will be used for searching hosts.
                     - Required if I(state=present).
-                type: str
+                type: list
```

Comment:
This should fix the last error that you are getting:
```suggestion
                type: list
                elements: str
```